src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/service/SignedOperationCapacityService.kt

```diff
@@ -31,6 +35,10 @@ class SignedOperationCapacityService(
   fun saveSignedOperationalCapacity(request: SignedOperationCapacityValidRequest): SignOpCapResult {
     var newRecord = true
 
+    val maxCap = locationService.getResidentialLocations(request.prisonId).prisonSummary?.maxCapacity ?: throw PrisonNotFoundException(request.prisonId)
+    if (maxCap < request.signedOperationCapacity) {
+      throw CapacityException(request.prisonId, "Signed operational capacity cannot be more than the establishment's maximum capacity of $maxCap", ErrorCode.SignedOpCapCannotBeMoreThanMaXCap)
+    }
```

Comment:
Can maxCap be equal to signedOperationCapacity ?

Reply:
Yes e.g. 12 < 12 = false